_posts/2023-10-21-beneath-the-shell.md

```diff
@@ -0,0 +1,65 @@
+
+<img class="centered" src="/images/blog/ka-het/super-sized.jpg" width="400" height="300" />
+
+## The Fetri'sei, and the exploration of deep space
+
+It was around this time that, between the various departments of the interstellar authority, the Ka'sei were nearing their peak: managing over two dozen stations in the core systems (with three more in various stages of construction) and a number smaller projects on the surface, they had grown far beyond the role they were supposed to have and made many inside the government uneasy about the power they could exert. To start breaking the hegemony that was starting to form the Ka'mar were created, intended to take some of the most promising space-based projects and develop them to completion - for instance, the then-new program of automating a large part of the interstellar traffic through new, robotic spacecrafts and surface bases. The Ka'sei's response didn't have to wait long: a competing program, which instead of automated spacecrafts put super-sized organisms at the helm and would have been capable of space travel, exploration and self replication; thus, the Fetri were born.
+
+<img class="centered" src="/images/blog/ka-het/fetri-sei-exploded.jpg" width="570" height="370" />
+
+With a lifetime measured in hundreds of years and an essentially infinite capability to replicate, it could achieve something only a few probes could ever do: exploring what's beyond the hyperlanes, going further than ever before at sublight speeds and report back everything that can be seen. Which it did, in some way. Nearly a century after the program's creation the first Ka'sei reached a neighbouring system, a g-type star disconnected from the hyperlane net, and was blown apart within seconds; its three sister probes, launched a few days behind the first, also quickly fell to enemy fire. The first result was panic; the second, however, was the clear realization of the need for a way to defend from the rest of the universe, and strike back to teach them their place, much more than simple warships could. The situation was dire, however.
+
+As the Fetri'sei began splitting between the ones reaching empty systems and the ones destroyed before replicating once, the central government started looking around for options; the Ka'mar, once touted to be on their way to become a juggernaut, had completely and publicly failed to deliver on the automation project, only completing a few subscale demonstrations and small surface facilities. Whether this was caused by sabotage or simple mismanagement is irrelevant; what counts is that this option was now unviable, making the choice of a super-sized organism-controlled warship obvious. Still anxious about the power of the Ka'sei, a new department was created for the purpose: the Ka'het, integrating the power systems and limited automation created by the Ka'mar over the years with the super-sizing technology.
```

Comment:
> I'm not really sure what is meant by this. What's bad about reaching an empty system? What the distinction being made that would cause the government to look for other options?

The focus wasn't supposed to be on the empty systems, I've reworded it to make it clearer